data/api/client-server/report_content.yaml

```
@@ -74,5 +77,13 @@ paths:
             type: object
           examples:
             application/json: {}
-      tags:
-        - Reporting content
+        404:
+          description: |-
+            The event was not found or you do not have permission to read this event.
```

MSC2249 has [a line](https://github.com/matrix-org/matrix-spec-proposals/blob/hs/reporting-fixes/proposals/2249-report-require-joined.md#proposal) that says it is up to implementations whether they would like to return this error for reported events that have been redacted. Could you add that here?

```suggestion
            The event was not found or you do not have permission to read this event.
            
            Homeserver implementations may additionally return this error if the
            reported event has been redacted.
```

---

without "may", and wording change:
```suggestion
            The event was not found or you are not joined to the room where the
            event resides.
            
            Homeserver implementations can additionally return this error if the
            reported event has been redacted.
```